headers/types/files/wan.h

```diff
@@ -38,7 +38,8 @@ struct wan_palettes {
     uint16_t unk1;
     uint16_t nb_color;
     uint16_t unk2;
-    uint16_t unk3;
+    uint8_t unk3;
+    uint8_t unk5;
```

Comment:
Is it really necessary to keep `unk4` as is? I would rather rename this one to `unk4` and `unk4` to `unk5` so that everything stays in order.

Reply:
Totally possible. It’s just that I thought it would avoid breaking potential (certainly non-existant) code that depend on unk4. I’ll do that if you think it’s better (I’m a bit undecided myself).